sample-app/src/components/DropdownSection.tsx

```diff
@@ -0,0 +1,108 @@
+import classNames from "classnames";
+import { useState, useEffect } from "react";
+
+export interface Option {
+  value: string,
+  display: JSX.Element
+}
+
+interface DropdownSectionProps<T> {
+  focusStatus?: string,
+  options: T[],
```

Comment:
We may not need a generic for the options if each option has a display prop

Reply:
Oh that's true, because it would be unlikely that any more information is needed for the Option other than the display and what value should be used for the input bar. I'll remove it